app/src/main/java/com/google/android/samples/socialite/ui/player/preloadmanager/PreloadManagerWrapper.kt

```diff
@@ -148,6 +153,7 @@ private constructor(
         defaultPreloadManager.release()
         preloadWindow.clear()
         mediaItemsList.toMutableList().clear()
+        playbackAndPreloadThread.quit()
```

Comment:
As this is shared by player and preload manager, quit the looper from the preload manager may not be safe. Shall we call this in the `releasePlayer()` in `TimelineViewModel.kt` after calling player release and preload manager release?

In the `TimelineViewModel.kt`, the `playbackAndPreloadThread` is a `HandlerThread` rather than `Looper`, and you can also call `playbackAndPreloadThread.quit()` because `HandlerThread.quit()` quits the handler thread's looper - then should do the same thing [1].

[1] https://developer.android.com/reference/android/os/HandlerThread#quit().